blocks/columns/columns.css

```diff
@@ -38,15 +38,14 @@
 
   .columns-wrapper {
     max-width: var(--section-desktop-max-width);
-    padding: 0 var(--section-desktop-padding);
-    margin: 0 auto;
+    padding: 0 var(--section-large-desktop-padding);
```

Comment:
Are you sure ? Seems like the opposite - start with large and go to small as screen increases ?

Reply:
I am sure :) default in original site is 
```
padding: 0 20px;
max-width: 999pt;
```
and the value is changing to 50px between 992-1599
```
@media (max-width: 1599px)
.we-container {
    max-width: 975pt;
    padding: 0 50px;
}

```

Reply:
Then shouldn't the variable values be interchanged ? And why only in columns ? There are other components using the exact same style

Reply:
I reverted css for columns